include/galsim/SBInclinedExponential.h

```diff
@@ -69,9 +69,12 @@ namespace galsim {
         /// @brief Returns the scale radius r0 of the disk profile
         double getScaleRadius() const;
 
-        /// @brief Returns the scale height r0 of the disk profile
+        /// @brief Returns the scale height h0 of the disk profile
         double getScaleHeight() const;
 
+        /// @brief Returns the scale height over scale radius h0/r0 of the disk profile
+        double getScaleHOverR() const;
```

Comment:
I'm not sure why this is useful.  What is the advantage of calculating this in the C++ layer?  It seems to me that the the C++ class is perfectly fine being defined in terms of just h and r.  

I don't even think we need the python-layer `getScaleHOverR` method, but even if you want to keep that, I definitely don't see any advantage to implementing it in C++.